pkg/asset/ignition/bootstrap/content/bootkube_temporary.go

```diff
@@ -43,6 +44,17 @@ spec:
     targetPort: 53
 `))
 
+	BootkubeKubeProxyKubeConfig = template.Must(template.New("kube-proxy-kubeconfig").Parse(`
+apiVersion: v1
+kind: Secret
+metadata:
+  name: kube-proxy-kubeconfig
+  namespace: kube-system
+data:
+  kubeconfig: {{ .AdminKubeConfigBase64 }}
```

Comment:
Do we need to use the admin kubeconfig here (vs. [the kubelet kubeconfig][1])?  It may not be a big deal either way if we're going to drop this in the next week or so.

[1]: https://github.com/openshift/installer/blob/ef04af3d3e92dbffcad13c00f445d53d1d3d6c9e/pkg/asset/kubeconfig/kubelet.go